includes/class-amp-post-template.php

```diff
@@ -62,7 +62,12 @@ public function get( $property, $default = null ) {
 	}
 
 	public function load() {
-		$this->load_parts( array( 'single' ) );
+
+		if( true === is_post_type_hierarchical( get_post_type( $this->ID ) ) ) {
```

Comment:
We can probably move this check to a helper function since it's repeated.

Alternate thoughts: I wonder if we should just be more explicit with both this and the metadata below. For the template we could introduce a hierarchy via a new function like `load_first_available_part( array( $this->post->post_type, 'single' ) )` where it finds and displays only the first available template from the array.

Similarly, for metadata, we default post => BlogPosting and page => WebPage and default everything else to a higher level schema object like `CreativeWork` or even `Thing`.

Let me know what you think.